src/defaults.rb

```diff
@@ -4,6 +4,11 @@
 
 
 module Metalware
+  # When new defaults are added here you may also want to add these to the
+  # option's help to be displayed.
+  # TODO: Could have these displayed in help automatically by overriding
```

Comment:
Thanks for reviewing - I think I'll just leave the TODO comment for now, it should be clear if we need to make the change in future, and no need to add cards for all potential future improvements :slightly_smiling_face:.